hvac/api/system_backend/mount.py

```diff
@@ -106,6 +106,25 @@ def disable_secrets_engine(self, path):
             url=api_path,
         )
 
+    def read_configuration(self, path):
```

Comment:
So this doesn't seem like a `/sys/mount` sort of method in particular so I don't know if it fits within this class's scope. I.e., while _some_ secrets engines have a `/config` route it isn't universal. Is there a particular set of secrets engines, etc. that you would use this method for? If so, I would say the method would better fit under those engines' classes.

Reply:
Sure then i will add this under the Active directory(secret engine) class
I read configuration to skip overriding configuration by reading the existing config